es-app/src/main.cpp

```diff
@@ -355,8 +355,8 @@ int main(int argc, char* argv[])
 		int deltaTime = curTime - lastTime;
 		lastTime = curTime;
 
-		// cap deltaTime
-		if((deltaTime > PowerSaver::getTimeout() && PowerSaver::getTimeout() > 0) || deltaTime < 0)
+		// cap deltaTime if it goes over non-zero Screensaver timeout or if negative
+		if((deltaTime > PowerSaver::getTimeout() + 100 && PowerSaver::getTimeout() > 0) || deltaTime < 0)
```

Comment:
We likely shouldn't be subtracting 100 in the first place, in PowerSaver::getTimeout().